onnx/checker.cc

```diff
@@ -151,8 +151,8 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& ctx) {
               "' points outside the directory");
         }
         std::wstring data_path = path_join(utf8str_to_wstring(ctx.get_model_dir()), relative_path);
-        struct _stat buff;
-        if (_wstat(data_path.c_str(), &buff) != 0) {
+        struct _stat64 buff;
+        if (_wstat64(data_path.c_str(), &buff) != 0) {
```

Comment:
Thanks for the fix. I wonder if we need a similar fix for the POSIX version below?

Reply:
+1 It seems useful and harmless that we can also change:
https://github.com/onnx/onnx/blob/30619cd473119c94088e20762089e54443cdac50/onnx/checker.cc#L187-L188
as `stat64` for non-Windows?